.eslintrc.js

```diff
@@ -1,7 +1,7 @@
 var fs = require("fs");
 let schema;
 try {
-  schema = fs.readFileSync("./sdlschema", "utf8");
+  schema = fs.readFileSync("./sdlschema.graphql", "utf8");
```

Comment:
added this so we only need one symlink for codegen and query linting

Reply:
Thanks!